assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -168,11 +173,11 @@ const Block = ( { shippingMethods = [], isEditor = false } ) => {
 								'woo-gutenberg-products-block'
 							) }
 							value={ shippingFields.country }
+							autoComplete="country"
 							onChange={ ( newValue ) =>
 								setShippingFields( {
 									...shippingFields,
 									country: newValue,
-									state: '',
```

Comment:
Is there any way to keep this? It was required to avoid the previous _State_ value being used when switching countries:
![Peek 2020-02-17 19-06](https://user-images.githubusercontent.com/3616980/74677440-97c00e80-51b8-11ea-80b8-5d34fa141e65.gif)

Reply:
Ill test again but I think it was removing the autocompleted value.

Reply:
NVM works, Ill put it back.